runtimevar/runtimeconfigurator/runtimeconfigurator.go

```diff
@@ -64,33 +56,23 @@ func Dial(ctx context.Context, ts gcp.TokenSource) (pb.RuntimeConfigManagerClien
 	return pb.NewRuntimeConfigManagerClient(conn), func() { conn.Close() }, nil
 }
 
-// A Client constructs runtime variables using the Runtime Configurator API.
-type Client struct {
-	client pb.RuntimeConfigManagerClient
-}
-
-// NewClient returns a new client that makes calls to the given gRPC stub.
-func NewClient(stub pb.RuntimeConfigManagerClient) *Client {
-	return &Client{client: stub}
-}
-
 // NewVariable constructs a runtimevar.Variable object with this package as the driver
 // implementation. Provide a decoder to unmarshal updated configurations into similar
 // objects during the Watch call.
-func (c *Client) NewVariable(name ResourceName, decoder *runtimevar.Decoder, opts *Options) (*runtimevar.Variable, error) {
-	w, err := c.newWatcher(name, decoder, opts)
+func NewVariable(name ResourceName, client pb.RuntimeConfigManagerClient, decoder *runtimevar.Decoder, opts *Options) (*runtimevar.Variable, error) {
```

Comment:
Similar advice here: move the client up in the parameter list since it is more general (i.e. could be shared by other calls to `NewVariable`) than the other parameters.

```suggestion
func NewVariable(client pb.RuntimeConfigManagerClient, name ResourceName, decoder *runtimevar.Decoder, opts *Options) (*runtimevar.Variable, error) {
```